app/assets/images/alert/alert-fail.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 96 96"><defs><style>.cls-1{isolation:isolate;}.cls-2{fill:url(#linear-gradient);}.cls-3{fill:url(#linear-gradient-2);}.cls-4{opacity:0.34;mix-blend-mode:multiply;fill:url(#linear-gradient-3);}.cls-5{fill:#820025;}.cls-6{fill:#fff;}</style><linearGradient id="linear-gradient" x1="20.07" y1="11.02" x2="72.88" y2="80.95" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ea4d67"/><stop offset="1" stop-color="#e21c3d"/></linearGradient><linearGradient id="linear-gradient-2" x1="38.56" y1="-5.51" x2="57.44" y2="101.51" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ea4d67"/><stop offset="1" stop-color="#a50025"/></linearGradient><linearGradient id="linear-gradient-3" x1="92.24" y1="94.99" x2="53.08" y2="52.78" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#969aa0" stop-opacity="0"/><stop offset="1" stop-color="#969aa0" stop-opacity="0.5"/></linearGradient></defs><title>alert-fail</title><g class="cls-1"><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><rect class="cls-2" width="96" height="96" rx="8" ry="8"/><path class="cls-3" d="M88,2a6,6,0,0,1,6,6V88a6,6,0,0,1-6,6H8a6,6,0,0,1-6-6V8A6,6,0,0,1,8,2H88m0-2H8A8,8,0,0,0,0,8V88a8,8,0,0,0,8,8H88a8,8,0,0,0,8-8V8a8,8,0,0,0-8-8Z"/><path class="cls-4" d="M16.26,53.39,57.36,96H88a8,8,0,0,0,8-8V58.75L79.64,41.81Z"/><circle class="cls-5" cx="87" cy="87" r="3"/><circle class="cls-5" cx="9" cy="87" r="3"/><circle class="cls-5" cx="87" cy="9" r="3"/><circle class="cls-5" cx="9" cy="9" r="3"/><path class="cls-6" d="M74.06,55.58H21.94a8,8,0,1,1,0-16H74.06a8,8,0,0,1,0,16Z"/></g></g></g></svg>
```

Comment:
Rename `alert/error-lg.svg` or `alert/fail-lg.svg` to match naming pattern of `warning-lg.svg`?